draft-ietf-dnsop-dns-catalog-zones.md

```diff
@@ -453,54 +458,42 @@ zones), but nameservers MUST NOT process such broken zones as catalog
-epoch property (see (#epochproperty)), all its associated state MUST be reset,
-including the zone itself.
-This can be relevant for example when zone ownership is changed.
-In that case one does not want the new owner to inherit the metadata.
-Other situations might be resetting DNSSEC state, or forcing a new zone transfer.
-This reset is tied to `<unique-N` because A simple removal followed by an addition of the member zone would be insufficient for this purpose because it is infeasible for secondaries to track, due to missed notifies or being offline during the removal/addition.
+## Member zone name clash {#nameclash}
 
-If an epoch property is present, the steps in (#epochproperty) describe how
-to signal a zone reset.
+If there is a clash between an existing zone's name (either from an existing member zone or otherwise configured zone) and an incoming
+member zone's name (via transfer or update), the new instance of the zone MUST
+be ignored and an error SHOULD be logged.
 
-# Migrating zones between catalogs
+A clash between an existing member zone's name and an incoming member zone's name (via transfer or update), may be an attempt to migrate a zone to a different catalog.
 
-[FIXME: clarify 2119 status of this and other properties and features. MUST? SHOULD? MAY? Because this one has security implications, also clearly note how ownership changes work without coo - namely, removing a member zone from catz A will cause catz B or C or D to pick up the zone on their next refresh. ]
 
-If there is a clash between an existing member zone's name and an incoming
-member zone's name (via transfer or update), this may be an attempt to do a Change Of Ownership.
+## Migrating member zones between catalogs {#zonemigration}
 
-A Change Of Ownership is signaled by the 'coo' property in the catalog zone currently owning the zone.
-The 'coo' property looks like "coo.<unique-N>.old.catalog PTR new.catalog".
+If all consumers of the catalog zones involved support the `coo` property, it is RECOMMENDED to perform migration of a member zone by following the procedure described in (#cooproperty).
+Otherwise a migration of member zone from a catalog zone `$OLDCATZ` to a catalog zone `$NEWCATZ` has to be done by: first removing the member zone from `$OLDCATZ`; second adding the member zone to `$NEWCATZ` after having made sure all catalog zone consumers of `$OLDCATZ` that are also consumer of `$NEWCATZ` had the member zone removed from `$OLDCATZ`.
 
-If there is no 'coo' property that resolves the clash, the zone remains owned by its current catalog and an error may be logged.
+If in the process of an migration some consumers of the involved catalog zones did not catch the removal of the member zone from `$OLDCATZ` (because of a lost packet or down time or otherwise) they may consider the update with the addition of the member zone in `$NEWCATZ` to be a name clash (see #nameclash) and as a consequence the member is not migrated to `$NEWCATZ`.
```

Comment:
This sentence does not make sense to me. If a consumer cannot notice that a member zone was removed from a catalog, that consumer has not kept enough state.

Reply:
ah - did not catch it *yet*. Perhaps you can add 'yet' :)